plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -552,6 +552,12 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
                                                         "retried_records_total",
                                                         "Total number of retried records.",
                                                         2, (char *[]) {"status", "name"});
+    /* Compress (gzip) */
+    tmp = flb_output_get_property("compress", ins);
```

Comment:
Couldn't find the "Unresolve conversation" button, so adding this here to flag that there was a follow-up comment in the earlier (resolved) thread.